Изменения в новую ветку

Ramilyushka opened the pull request.

Ramilyushka committed:
Изменения в новую ветку

Ramilyushka committed:
Верстка ячейки

Ramilyushka committed:
Заполнены 20 ячеек картинками и кнопками лайк/дизлайк и дата в метод configCell

andrew-kuzn-yndx commented on ImageFeed/ImagesList/ImagesListViewController.swift:
```diff
@@ -0,0 +1,81 @@
+        guard let image = UIImage(named: photoNames[indexPath.row]) else {
+            return 0
+        }
+        let imageInsets = UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16)
```
Наверное, боковые отступы тут лишние, так как мы уже задали их для tableView

andrew-kuzn-yndx commented on ImageFeed/ImagesList/ImagesListViewController.swift:
```diff
@@ -0,0 +1,81 @@
+
+extension ImagesListViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    }
```
Выделение ячеек нам не нужно, поэтому этот метод делегата можно убрать)

Ramilyushka committed:
Поправила замечания ревью:
1) Убрала отступы по бокам в констрейнтах
2) Убрала комментарии и лишние отступы

Ramilyushka committed:
Убрала лишний метод делегата "выделение ячеек"

Ramilyushka committed:
Поправила констрейнты
